cinema/serializers.py

```diff
@@ -12,6 +12,15 @@
 )
 
 
+class MovieImageSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = (
+            "id",
+            "image",
+        )
+
```

Comment:
Clean Code: The `MovieImageSerializer` class seems to be unused in this file. If it's not used anywhere else in your project, consider removing it to keep your codebase clean.

Reply:
cinema/views 107 row